com.unity.render-pipelines.core/Runtime/PostProcessing/Shaders/LensFlareCommon.hlsl

```diff
@@ -96,7 +96,7 @@ float GetOcclusion(float2 screenPos, float flareDepth, float ratio)
 
     for (uint i = 0; i < (uint)_OcclusionSampleCount; i++)
     {
-        float2 dir = _OcclusionRadius * SampleDiskUniform(Hash(2 * i + 0 + 1), Hash(2 * i + 1 + 1));
+        float2 dir = _OcclusionRadius * SampleDiskUniform(Hash(2 * i + 0), Hash(2 * i + 1));
```

Comment:
Any reason why we have `+ 0` there?

Reply:
Yes, to make the code more readable, that have no perf impact.

Reply:
uhhh, okay. Gonna have to disagree on the readability aspect here.